sdk/tests/conformance/rendering/framebuffer-switch.html

```diff
@@ -42,6 +42,7 @@
 "use strict";
 description("Test framebuffer switching. The test switches between two framebuffers, copying rendering results from one to the other.");
 var wtu = WebGLTestUtils;
+var canvas = document.getElementById("canvas");
 
 var gl = wtu.create3DContext("canvas");
```

Comment:
Could this have been changed to:
create3DContext(canvas)

instead of "canvas", at this point?

Only a minor question.